README.md

```diff
@@ -44,6 +46,15 @@ permitted_domains: 'hashrocket.com|hshrckt.com'
 permitted_emails: 'friend@whitelist.com'
 ```
 
+##### To get admin route working on localhost:
+
+```
+1. Navigate to https://console.developers.google.com/
```

Comment:
Nice find; that article seems helpful. As does the README for the gem we use: https://github.com/zquestz/omniauth-google-oauth2/blob/master/README.md. It seems to cover the proposed setup steps in even greater detail.

I agree that authentication is under-documented in the project. However, anytime we document another service's UI, I get nervous, because they can (and do, in Google's case) change their UI all the time. A checklist that is wrong is worse than no checklist, in my opinion.

I think the solution here is something like this, in the part of the README that already talks about authentication (Install):

```
Authentication is managed by Omniauth and Google. See the [omniauth-google-oauth2  README](https://github.com/zquestz/omniauth-google-oauth2/blob/master/README.md) and [Google Ouath 2 docs](https://developers.google.com/identity/protocols/OAuth2WebServer) for setup instructions.
```